specs/http-client2/index.html

```diff
@@ -255,14 +285,23 @@ <h3>Request Options</h3>
         </p>
 
         <dl>
-          <dt><code>status-only</code> (<code>xs:boolean</code>)</dt>
-          <dd>Skip body in response. Default: false.</dd>
+          <dt><code>http-version</code> (<code>xs:string</code>)</dt>
```

Comment:
Maybe we can simply raise `http:options`, as specified at the bottom of the section (otherwise, we’d need to handle all the corner cases, such as negative timeout values, header field names with invalid characters, etc.

Reply:
I think I understand. When talking about “an unsupported version”, we may have two different cases:

1. The specified value is invalid, simply because it’s not of type xs:string, or a string different to `1.0` or `1.1`. In this case (i.e., if the error is caused while parsing the XQuery request), I would raise `http:options`.
2. We are sending a 1.1 request, which does not support 1.1. Instead of returning a result, we could raise an error here. So I guess you mean this case? What’s the best way to detect that the server does not support a given HTTP version?

Reply:
Ah okay. Sorry @ChristianGruen I think I have set out to confuse you completely here by accident. I thought your comment was on `http-version` in the response and not the request.

I also missed the "raise" part of your suggestion. You are suggesting an error code named `http:options`. For some reason (too much Sun?) I though you were suggesting to do a function call to `http:options` in the event of a bad response. 

Perhaps we could name the error code `http:invalid-options` to avoid my confusion again ;-)